lib/checkunusedfunctions.cpp

```diff
@@ -248,10 +291,12 @@ void CheckUnusedFunctions::check(ErrorLogger * const errorLogger, const Settings
             it->first == "if")
             continue;
         if (!func.usedSameFile) {
-            std::string filename;
-            if (func.filename != "+")
-                filename = func.filename;
-            unusedFunctionError(errorLogger, filename, func.lineNumber, it->first);
+            if (!isOperatorFunction(it->first)) {
+                std::string filename;
+                if (func.filename != "+")
+                    filename = func.filename;
+                unusedFunctionError(errorLogger, filename, func.lineNumber, it->first);
```

Comment:
so if func.filename is "+" then no filename is written for unusedFunctionError? is this intended? Maybe that is how it should work I don't remember.